website/docs/faq.md

```diff
@@ -101,6 +101,9 @@ bundle exec srb init
 
 # For plugins like sorbet-rails, see their docs, eg.
 https://github.com/chanzuckerberg/sorbet-rails#initial-setup
+
+# Suggest new, hopefully stronger sigils (per-file strictness levels)
```

Comment:
I like this, but could you reword slightly to say that you may get more errors at a higher typed level? We've seen problems on slack from automatically updating the sigils, so it would be great to set expectations with this message.

Reply:
I didn't know that `suggest-typed` was fallible. I thought it attempts stronger and stronger sigils until it encounters an error, then backs off. So, I'm unsure what to write. Can you please suggest the wording you prefer?

Reply:
Sorry, I think that I was crossing my wires on the problems with `suggest-typed` that I was seeing recently. The problems that people were reporting were cases where the typed level was downgraded, despite there being no type errors present. Perhaps saying something like, "the suggestion process is fallible, and may suggest downgrading when it's not necessary" would be good.

Reply:
> Perhaps saying something like ..

Updated accordingly, thanks Trevor